MNG-76 Create Loading Page

memeraki opened the pull request:
styled loading page
added fonts
addes variables - colors and some mixins

memeraki committed:
MNG-76 Create Loading Page

styled loading page
added fonts
addes variables - colors and some mixins

lukaszdutka commented on styles/SASS/_config.scss:
```diff
@@ -1,13 +1,35 @@
-$background-color: white;
-$text-color: black;
-$mobile: 800px;
+$color-red: #9b1d20ff;
+$color-black: #020402ff;
+$color-white: #e8f1f2ff;
+$color-blue: #235789ff;
+$color-yellow: #e3b505ff;
```
maybe rename it with some names like "color-primary", "color-text" etc. Other devs might don't know when to use which color.
Or maybe add alias for it? 🤷

AleksandraCyp replied:
I think that the way Gosia made it is better, because we may mix these colors and not always use them for the same purpose.

memeraki replied:
we got two text-colors (black and white depends on the background-color) and three primary colors: red for the start view and blue or yellow for the fight view (depends on the current player)

mariusz-sm replied:
It is good idea to add "color-primary" and "color-text" names.  It could help in code reading.

AleksandraCyp commented on index.html:
```diff
@@ -6,12 +6,21 @@
+        <p>Please wait a bit, we are looking for the Poké Flute to wake up Snorlax to fight.</p>
+        <img id="zzz" src="assets/zzz.png" alt="zzz">
+        <img src="assets/snorlax.png" alt="sleeping snorlax">
+    </div>
```
Shouldn't it be a div with the id "app" in which we will then load different templates?

memeraki replied:
I thought about it. This is the start screen we see before our application loads (similar to our names in the pokemon quiz). 
I was thinking about create in html body two divs: first with class or id equal "content" or "app" to load templates and second "footer" with only discription as u see here. But i'm not sure where to put this loading view. In my opinion this current view shouldnt be a template.    
What do you think about this solution for html body:
<body>

    <div id="app">
        <div id="loading-view">
            <p>Please wait a bit, we are looking for the Poké Flute to wake up Snorlax to fight.</p>
            <img id="zzz" src="assets/zzz.png" alt="zzz">
            <img src="assets/snorlax.png" alt="sleeping snorlax">
        </div>
        <!-- below div for others views -->
        <template id="current-view"></template>
    </div>
    
    <footer>
        TypeScript Mangos project for CodersCamp2020
    </footer>

    <script src="dist/index.js"></script>

</body>

AleksandraCyp replied:
```suggestion
<div id="app">
    <div id="loading-view">
        <p>Please wait a bit, we are looking for the Poké Flute to wake up Snorlax to fight.</p>
        <img id="zzz" src="assets/zzz.png" alt="zzz">
        <img src="assets/snorlax.png" alt="sleeping snorlax">
    </div>
</div>

<footer>
    TypeScript Mangos project for CodersCamp2020
</footer>

    <!-- below div for others views -->
    <template id="current-view"></template>

<script src="dist/index.js"></script>
```
I like your suggestion! We could also put the templates outside of the app div (like we did last time and like I suggested aboce), but I'm not sure if it would make any difference, so maybe it sould be up to you to decide :)

mariusz-sm replied:
I agree with @memeraki . Loading view won't be a template but we will use templates for other views.

memeraki committed:
MNG-76 loading page

fixed variable names
added transition
added template placeholders for diffrent views to html
